Packs/DuoAdminApi/Integrations/DuoEventCollector/DuoEventCollector.yml

```
@@ -4,26 +4,18 @@ commonfields:
 name: Duo Event Collector
 display: Duo Event Collector
 category: Analytics & SIEM
-description: Duo logs event collector integration for XSIAM.
+description: Collects Auth and Audit events for Duo using the API(https://duo.com/docs/adminapi#logs).
```

you can add the link to the API in the README.
```suggestion
description: Collects Auth and Audit events for Duo.
```